features-json/css-anchor-positioning.json

```diff
@@ -704,7 +704,8 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Can be enabled via the `#enable-experimental-web-platform-features` flag in `chrome://flags`"
+    "1":"Can be enabled via the `#enable-experimental-web-platform-features` flag in `chrome://flags`",
+    "2": "There's a [bug with `position: fixed` and non-initial containing blocks](https://codepen.io/pavelp/pen/KwVPrrB)."
```

Comment:
I'm not sure if a link to a personal codepen is appropriate here, so I'm happy to use it as a conversation starter and discuss what a more durable alternative could be if this change is overall acceptable.